boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0.dts

```diff
@@ -43,15 +43,15 @@
 		compatible = "gpio-keys";
 		user_button_1: button_0 {
 			label = "User SW1";
-			gpios = <&gpio0 5 GPIO_ACTIVE_HIGH>;
+			gpios = <&gpio0 5 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
```

Comment:
Is this change really something for this patch?

Reply:
It's present because the LPCxpresso55s69 board `pinmux.c` file previously was setting the GPIO buttons to pull up in an initialization function. Without that pinmux call, these flags need to be added so that the gpio driver is aware these inputs should be pulled up.

Reply:
Ah, gotcha, thanks for the explanation. Resolved. Might want to add to the commit log but not a blocker.